less/navigation.less

```diff
@@ -1 +1,43 @@
-// Navigation code here
\ No newline at end of file
+// Navigation code here
+.grey-header {
+  width: 100%;
+  background-color: @nav-bg;
+  color: @header-font-color;
+  position: fixed;
+  top: 0;
+
+  @media @phone{
+    position: unset;
+    width: 100%;
+  }
+
+  .title-heading {
+    font-size: 4rem;
+    text-align: center;
+
+    @media @phone {
+      padding: 3% 0 3% 0;
+    }
+
+  }//I wanted to use calc() to make this font responsive because I had seen it on CSS tricks but I couldn't find the page again, I really wanted this title heading to grow and shrink with the size of the viewport.
```

Comment:
If you want something based off of the viewport, there is the vh and vw units.  I generally try to avoid those though.